docs/aks-installation.md

```diff
@@ -53,26 +53,26 @@ You can use any method to create an AKS cluster. The example given here is using
     kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-aks-node-init.yml
     ```
 
-2. Download Antrea YAML
-
-    Deploy a released version of Antrea from the [list of releases](https://github.com/vmware-tanzu/antrea/releases).
-Note that AKS support was added in release 0.9.0, which means you cannot pick a release older than 0.9.0.
-For any given release `<TAG>` (e.g. `v0.9.0`), get the Antrea AKS deployment yaml at:
-
-    ````
-    https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-aks.yml
-    ````
-
-    To deploy the latest version of Antrea (built from the master branch) to AKS, get the Antrea AKS deployment yaml at:
-
-    ````
-    https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-aks.yml
-    ````
-
-3. Deploy Antrea
-
-    Deploy Antrea using `kubectl apply -f antrea-aks.yml`. It will deploy a single replica of Antrea controller to the AKS cluster
-and deploy Antrea agent to every Node. After a successful deployment you should be able to see these Pods running in your cluster:
+2. Deploy Antrea
+
+    To deploy a released version of Antrea, pick a deployment manifest from the
+[list of releases](https://github.com/vmware-tanzu/antrea/releases).
+Note that AKS support was added in release 0.9.0, which means you can not
+pick a release older than 0.9.0. For any given release `<TAG>` (e.g. `v0.9.0`),
+you can deploy Antrea as follows:
+```bash
```

Comment:
looking at the rendered Markdown file, the code blocks here are not consistent with the ones above. Could you remove the indentation for the ones above as well. May apply to the other cloud files modified in this PR as well.

Reply:
You are right. I added intents for all command blocks.